bridge-adaptivity/bridge_lti/urls.py

```diff
@@ -5,7 +5,6 @@
 from .consumer import source_preview
 
 urlpatterns = [
-    url(r'^launch$', lti_launch, name='initial-launch'),
-    url(r'^launch/(?P<collection_id>\d+)$', lti_launch, name='launch'),
+    url(r'^launch/?(/(?P<collection_id>\d+))?/?$', lti_launch, name='launch'),
```

Comment:
This pattern looks a little bit ugly: why don't use something like this: `r'^launch/(?P<collection_id>\d*)$'` ?